INSTALL.md

```diff
@@ -2,7 +2,7 @@
 
 This package can be installed from PyPI, or by cloning the repository, depending on your use-case.
 
-1. To use the `optimade` Python package as a library, (e.g., using the models for validation, parsing filters with the grammar, or using the command-line tool `optimade-validator` tool), it is recommended that you install the latest release of the package from PyPI with `pip install optimade`.
+1. To use the `optimade` Python package as a library, (e.g., using the models for validation, parsing filters with the grammar, or using the command-line tool `optimade-validator` tool), it is recommended that you install the latest release of the package from PyPI with `pip install optimade`. If you also want to use the OPTIMADE client to query OPTIMADE APIs, you should install with the additional dependencies: `pip install optimade[http_client]`.
```

Comment:
Yes, this would mean we use a separate package. However, the dependencies for the optimade PyPI are not large by default, and it is useful to re-use some code between them.